src/python/pants/backend/python/subsystems/python_native_code.py

```diff
@@ -13,21 +13,22 @@ class PythonNativeCodeSubsystem(Subsystem):
     options_scope = "python-native-code"
```

Comment:
For posterity: this subsystem has several problems, including https://github.com/pantsbuild/pants/issues/14612

I don't think it blocks this PR, given that environment targets are still experimental. But we should probably fix that ticket before 2.15